NDTensors/ext/NDTensorsMetalExt/linearalgebra.jl

```diff
@@ -1,16 +1,30 @@
-function NDTensors.qr(leaf_parenttype::Type{<:MtlArray}, A::AbstractMatrix)
-  Q, R = NDTensors.qr(NDTensors.cpu(A))
-  return adapt(leaf_parenttype, Matrix(Q)), adapt(leaf_parenttype, R)
+function LinearAlgebra.qr(A::Exposed{<:MtlMatrix})
+  Q, R = LinearAlgebra.qr(expose(NDTensors.cpu(A)))
```

Comment:
Maybe we should use `using LinearAlgebra` so you can just use `qr` instead of `LinearAlgebra.qr`.